csharp/src/Apache.Arrow/Ipc/ArrowFileReaderImplementation.cs

```diff
@@ -36,7 +36,7 @@ internal sealed class ArrowFileReaderImplementation : ArrowStreamReaderImplement
         /// <summary>
         /// Notes what byte position where the footer data is in the stream
         /// </summary>
-        private int _footerStartPostion;
+        private long _footerStartPostion;
```

Comment:
We actually don't use the field for anything. We can remove the instance field and just make it a local variable in the functions that use it today.